ivy/functional/frontends/torch/tensor_functions.py

```diff
@@ -43,4 +45,4 @@ def is_nonzero(input):
         1,
         message="bool value of tensor with more than one or no values is ambiguous",
     )
-    return Tensor(ivy.nonzero(input)[0].size != 0)
+    return Tensor(ivy.nonzero(input)[0].size != 0)
```

Comment:
Seems to be that you dont have a precommit hook installed, you can read up about that [here](https://lets-unify.ai/ivy/contributing/setting_up.html#pre-commit)